enterprise/frontend/src/embedding/data-picker/DataSelector/DataSelector.jsx

```diff
@@ -695,13 +632,6 @@ export class UnconnectedDataSelector extends Component {
     await this.nextStep({ selectedTableId: table?.id });
   };
 
-  onChangeField = async (field) => {
```

Comment:
`onChangeField` was only used in `enterprise/frontend/src/embedding/data-picker/DataSelectorFieldPicker/DataSelectorFieldPicker.tsx` but that file is also removed in this PR.